micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/ProductDetails/ProductDetailsComponent.js

```
@@ -0,0 +1,114 @@
+                    ) 
+                    : item.type === "step-heading" ? (
+                      <CardText key={itemIndex} className="custom-step-header">
+                        {t(item.text)}
+                      </CardText>
+                    ) : item.type === "step" ? (
+                      <li key={itemIndex} className="custom-step-item">
+                        {t(item.text)}
+                      </li>
+                    ) : item.type === "image" ? (
+                      <div key={itemIndex} className="custom-image-container">
+                        <img src={item.text} alt={module + "Image"} className="custom-image" />
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)